xbmc/pvr/windows/GUIWindowPVRTimers.cpp

```diff
@@ -223,6 +224,13 @@ bool CGUIWindowPVRTimers::OnMessage(CGUIMessage &message)
         Update(GetDirectoryPath());
         bReturn = true;
       }
+      else if (message.GetSenderId() == CONTROL_BTNHIDEDISABLEDTIMERS)
+      {
+        CSettings::GetInstance().ToggleBool(CSettings::SETTING_PVRTIMERS_HIDEDISABLEDTIMERS);
```

Comment:
Only from the sidebar, as settings level is 4. => https://github.com/xbmc/xbmc/pull/8543/files#diff-92b91a5c902936f6e9a03f26b054d7d6R1637

The settings in addon manager confluence sidebar (like "Hide incompatible") behave exactly like this. I actually "borrowed" the implementation ghist from there.